### ISSUE TITLE
Wrist pain subtypes

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10755]
**Preferred term label:**
4 types of wrist pain distinctions are used by orthopedists:

Radial-sided wrist pain
Ulnar-sided wrist pain
Volar wrist pain
Dorsal wrist pain

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0030836

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Radial:
Intersection syndrome of hand tendons
De Quervain tenosynovitis

Ulnar:
Extensor carpi ulnaris (ECU) tendonitis
Ulnocarpal abutment syndrome

Volar:
Flexor carpi radialis (FCR) tendonitis
Scaphoid nonunion advanced collapse (SNAC)

Dorsal:
Intersection syndrome of hand tendons
Extensor digitorum communis tenosynovitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
[Ulnar wrist pain HP:6001200](https://hpo.jax.org/browse/term/HP:6001200)

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001404
- label: Radial wrist pain
- parents: [Wrist pain (HP:0030836)](https://hpo.jax.org/app/browse/term/HP:0030836)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) localized to the radial side of the wrist.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10755
- synonyms: Radial-sided wrist pain(NONE)

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001405
- label: Volar wrist pain
- parents: [Wrist pain (HP:0030836)](https://hpo.jax.org/app/browse/term/HP:0030836)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) localized to the volar side of the wrist (i.e., to the palm side - front - of the wrist).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10755
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001406
- label: Dorsal wrist pain
- parents: [Wrist pain (HP:0030836)](https://hpo.jax.org/app/browse/term/HP:0030836)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) localized to the dorsal side (back) of the wrist.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10755
- synonyms:
